packages/react-server-dom-webpack/src/__tests__/ReactFlightDOM-test.js

```diff
@@ -841,6 +841,10 @@ describe('ReactFlightDOM', () => {
         expectedGamesValue,
     );
 
+    if (gate(flags => flags.enableOwnerStacks)) {
+      // TODO: assertConsoleServerErrorDev
```

Comment:
Probably best to use a dedicated `assertConsoleBadgedErrorDev` matcher with the badge name. That way we could also test switching environment names halfway through.